src/neptune/api/requests_utils.py

```diff
@@ -21,7 +21,7 @@
     Dict,
 )
 
-from requests.exceptions import JSONDecodeError
+from requests.compat import JSONDecodeError
```

Comment:
How about 
```
try:
   from requests.exceptions import JSONDecodeError
except ImportError:
   from requests.compat import JSONDecodeError
```
?
Should be a little more readable - we want to underscore that we are using exceptions from requests library. `compat` doesn't tell much about the nature of the package.
What do you think?